asset/testnet/app.toml

```diff
@@ -39,6 +39,7 @@ FixZeroBalanceHeight = 1
 BEP70Height = 1
 BEP67Height = 1
 BEP8Height = 1
+BEP153Height = 30516226
```

Comment:
Can we move it below `BEP151Height = 28250000` ?

Reply:
Nice catch